src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -630,25 +649,6 @@ protected function processCache($cache, $suffix)
 
 	public function beforeCompile()
 	{
-		$eventsExt = NULL;
-		foreach ($this->compiler->getExtensions() as $extension) {
-			if ($extension instanceof Kdyby\Events\DI\EventsExtension) {
-				$eventsExt = $extension;
-				break;
-			}
-		}
-
-		if ($eventsExt === NULL) {
-			throw new Nette\Utils\AssertionException('Please register the required Kdyby\Events\DI\EventsExtension to Compiler.');
-		}
-
-		$this->processRepositories();
-	}
-
-
-
-	protected function processRepositories()
```

Comment:
No problem. It's just that beforeCompile will only call this method and nothing else which seems a bit odd. But sure. :-)